features/features.go

```diff
@@ -459,4 +459,10 @@ var (
 						productScope(ocpSpecific).
 						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						mustRegister()
+
+	FeatureGateMachineAPIMigration = newFeatureGate("MachineAPIMigration").
```

Comment:
intentionally never enabled?

Reply:
Yes. As this is greenfield, there's a lot to build before we have something that works. My hope was that we can get this all set up to a point where there's actually something tangible before adding to a feature set some time in 4.17